### PR TITLE
Remove intermittent failure reporting.

### PR DESCRIPTION
**Pull request by jdm**
The frequency with which we hit https://github.com/servo/intermittent-failure-tracker/issues/5 means that this is hurting us more than it helps.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/24359)

### CODE REVIEW

**Comment by CYBAI** [comment]
```
Error running mach:

    ['filter-intermittents', 'wpt-errorsummary.log', '--log-intermittents', 'intermittents.log', '--log-filteredsummary', 'filtered-wpt-errorsummary.log', '--tracker-api', 'default', '--reporter-api', 'default']

The error occurred in the implementation of the invoked mach command.

This should never occur and is likely a bug in the implementation of that
command. Consider filing a bug for this issue.

If filing a bug, please include the full output of mach, including this error
message.

The details of the failure are as follows:

NameError: global name 'reported' is not defined
```

File: python/servo/testing_commands.py [hunk 1]
```diff
@@ -534,13 +534,6 @@ def filter_intermittents(self, summary, log_filteredsummary, log_intermittents,
                 else:
                     intermittents += [failure]
 
-        if reporter_api:
-            if reporter_api == 'default':
-                reporter_api = "https://build.servo.org/intermittent-failure-tracker"
-            if reporter_api.endswith('/'):
-                reporter_api = reporter_api[0:-1]
-            reported = set()
```
Comment (CYBAI) [inline python/servo/testing_commands.py]: maybe we can't remove this one?

Comment (jdm) [reply python/servo/testing_commands.py]: Argh.

**Comment by CYBAI** [comment]
```
Checking files for tidiness...

[94m./python/servo/testing_commands.py[0m:[93m17[0m: [91mF401 'platform' imported but unused[0m

[94m./python/servo/testing_commands.py[0m:[93m23[0m: [91mF401 'urllib' imported but unused[0m

[94m./python/servo/testing_commands.py[0m:[93m543[0m: [91mF841 local variable 'pull_request' is assigned to but never used[0m
```

**Pull request merged**